docs/tutorials/exoscale.md

```diff
@@ -48,7 +47,6 @@ spec:
         - --domain-filter={{ my-domain }}
         - --policy=sync # if you want DNS entries to get deleted as well
         - --txt-owner-id={{ owner-id-for-this-external-dns }}
-        - --exoscale-endpoint={{ endpoint }} # usually https://api.exoscale.ch/dns
```

Comment:
Add documentation for the optional settings for environment and zone?

Reply:
Our API client library ([egoscale](https://github.com/exoscale/egoscale)) will build the API endpoint URL using the provided zone and environment. 

Zone defines a datacenter location and will be used to build endpoint URL for API requests as described in our [API documentation](https://openapi-v2.exoscale.com/#topic-zone-local-resources). Up-to-date list of zones can be find [here](https://community.exoscale.com/documentation/platform/exoscale-datacenter-zones/) where `slug` values are used in API context.

Environment can be used to select pre-production (staging) version of our platform.

Reply:
Added a note about zone/environment in tutorial.